### PR TITLE
WIP: New index.html

### PR DESCRIPTION
**Pull request by Greggless**
fixes: https://github.com/ubuntudesign/web-squad/issues/890

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,227 +1,326 @@
...
+                  </div>
+              </li>                                                                           
+          </ul>
         </div>
-      </div>
-      <div class="row mt centered mobile-only">
-        <div class="col-lg-12">
+      </section>
+
+      <section><a name="quick-start"></a>
```
Comment (steverydz) [inline index.html]: I think it's better to put an ID on the `section` element here rather than an empty link

**Comment by ktsakalozos** [comment]
We recently (last week) got microk8s certified (https://www.cncf.io/certification/software-conformance/).

We can proudly wear the "certified" badge, like we do in the github README https://github.com/ubuntu/microk8s

**Comment by anasereijo** [comment]
I reviewed the page with @Greggless and we have some things to change/to do.

#### @anasereijo  list:

- [ ] create and save grey background suru

#### @inuwa  list:

- [ ] set microk8s logo to light
- [ ] set microk8s background colour to ubuntu orange (like we have on ubuntu.com)
- [ ] vertically align link and button
- [ ] remove blue style from p-list_item is-ticked 
- [ ] set a scrolling animation Get started button 
- [ ] code block: align code to the left side
- [ ] add footer from copy doc
- [ ] make sure it is responsive

**Comment by Greggless** [comment]
Closing as @inuwa will work his magic with the code.

Please refer to the [copy doc](https://docs.google.com/document/d/1ObWmhRc4GCIhrBi8gT7yIIs0J_dn_8ravfLtcwhr3uQ/edit#heading=h.rwj72ucjcwr4) and to [Ana's comment](https://github.com/canonical-websites/microk8s.io/pull/24#issuecomment-440278586) when building it for real

**Comment by Lukewh** [comment]
"There's inconsistent use of MicroK8s and microk8s."
_- An anonymous source_

**Pull request closed without merging**